binancial/data/get_klines_realtime.py

```
@@ -48,30 +71,57 @@ def handle_socket_message(msg):
+                kline_start_time,
+                kline_close_time,
+                first_trade_id,
+                last_trade_id,
+                open_price,
+                close_price,
+                high_price,
+                low_price,
+                base_asset_volume,
+                quote_asset_volume,
+                no_of_trades
+            ])
```

Ruff (0.8.2)

Consider f-string instead of string join

Replace with f-string

(FLY002)